src/backend/base/langflow/components/vectorstores/AstraDB.py

```diff
@@ -116,6 +116,7 @@ class AstraVectorStoreComponent(LCVectorStoreComponent):
             name="embedding",
             display_name="Embedding or Astra Vectorize",
             input_types=["Embeddings", "dict"],
+            info="Allows either an embedding model or an Astra Vectorize configuration. If Astra Vectorize is already configured for the collection, this field is not required.",
```

Comment:
did you try what happens if you don't set it ? 
I believe the AstraDBVectorStore requires on of those.

Reply:
Ugh, yeah. I guess that's something to figure out. It's required by langchain-astradb to have one or the other at the moment.

Reply:
Perhaps we could loosen that constraint iff the setup mode is `SetupMode.OFF`?